package.json

```diff
@@ -139,8 +140,5 @@
     "tslint-react": "^1.0.0",
     "webpack-dev-server": "^1.15.0",
     "yargs": "^4.8.0"
-  },
-  "pre-commit": [
```

Comment:
I prefer to keep it in, since you can just ignore it with `-n` if you are certain you don't want the lint checking, seems like a pain update files afterwards, but your call.